android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/Logger.java

```diff
@@ -0,0 +1,40 @@
+package com.shopify.unity.buy.utils;
+
+import android.util.Log;
+
+/**
+ * @author Flavio Faria
+ */
+
+public final class Logger {
```

Comment:
I have a couple questions about this class:

1. Was this class introduced so we don't have to reference the Android logger for unit tests since it's not stubbed?
2. Does it make sense to disable logging only when under test? I could see us always wanting to log errors for example. If there is additional things like info/debug logs that we want to include maybe use a verbose flag?

Reply:
1. Yes. Another reason is to be able to externally enable/disable logs as we wish.
2. The only thing I know is that we (and the 3rd-parties) probably don't want to keep logging enabled in production. At first, the only needs I had in mind now for this class was the ones described in 1., but we can definitely add more control to it.